<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-EKQQF59L" name="Random Sanity Test Demo" tests="2" failures="2" timestamp="28 Mar 2017 09:28:57 GMT" time="446.564" errors="0">
  <testcase name="VerificationDuringCall" time="181.487" classname="testCases.Framework_001">
    <failure type="java.lang.Exception">
      <![CDATA[java.lang.Exception
	at utility.Utils.takeScreenshot(Utils.java:99)
	at testCases.Framework_001.VerificationDuringCall(Framework_001.java:165)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- VerificationDuringCall -->
  <testcase name="VerificationBeforeCall" time="147.891" classname="testCases.Framework_002">
    <failure type="java.lang.Exception">
      <![CDATA[java.lang.Exception
	at utility.Utils.takeScreenshot(Utils.java:99)
	at testCases.Framework_002.VerificationBeforeCall(Framework_002.java:50)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- VerificationBeforeCall -->
</testsuite> <!-- Random Sanity Test Demo -->
